bob/src/cleaner.rs

```diff
@@ -52,19 +101,7 @@ impl Cleaner {
                 let hard = hard.unwrap_or(10);
                 backend.close_unneeded_active_blobs(soft, hard).await;
             }
-            if let Some(limit) = index_memory_limit {
-                let mut memory = backend.index_memory().await;
-                info!("Memory before closing old active blobs: {:?}", memory);
-                while memory > limit {
-                    if let Some(freed) = backend.close_oldest_active_blob().await {
-                        memory = memory - freed;
-                        debug!("closed index, freeing {:?} bytes", freed);
-                    } else {
-                        break;
-                    }
-                }
-                info!("Memory after closing old active blobs: {:?}", memory);
-            }
+            cleaner.request_index_cleanup().await;
             if let Some(limit) = bloom_filter_memory_limit {
                 backend.offload_old_filters(limit).await;
```

Comment:
An additional question to investigate: how does loading/offloading index affect bloom filters? Does index loading restore offloaded bloom filter? Will this restored bloom filter show up in the bloom filter hierarchy and be unloaded by calling `offload_old_filters`?

Reply:
1. Index loading restore offloaded bloom filter from disk.
2. Yes, they will be freed as filters cleanup work on all blobs, not only active.